Dockerfile.release

```diff
@@ -5,17 +5,26 @@ ARG BUILDPLATFORM
 
 WORKDIR /var/lib/avail-light
 
-ARG AVAIL_TAG=v1.7.8
-ARG AVAIL_LC_BIN=https://github.com/availproject/avail-light/releases/download/$AVAIL_TAG/avail-light-linux-$TARGETARCH.tar.gz
+ARG AVAIL_TAG=v1.12.0-rc2
+
+RUN apt-get update && apt-get install -y --no-install-recommends gettext wget ca-certificates \
```

Comment:
This introduces another layer in the container image, bloating the image size in the end.

Ideally we should use less `RUN` steps as possible